spritepacks/released/briaryoung/briaryoung-button-up-flowered/mod_assets/monika/j/briaryoung_clothes_button_up_flowered.json

```diff
@@ -0,0 +1,73 @@
+{
+    "__author": "Briar Young",
+    "ex_props": {
+        "bare-shoulders": true,
```

Comment:
this outfit does not actually have bare shoulders, so I don't think this should have this exprop. Is this here for hair compatibility purposes?